src/components/app/ProfileFilterNavigator.js

```diff
@@ -70,8 +70,18 @@ class ProfileFilterNavigatorBarImpl extends React.PureComponent<Props> {
       );
     } else {
       firstItem = (
-        <Localized id="ProfileFilterNavigator--full-range">
-          Full Range
+        <Localized
+          id="ProfileFilterNavigator--full-range-with-duration"
+          vars={{
+            fullRangeDuration: getFormattedTimeLength(
```

Comment:
In the future I'd like to replace this with a Fluent function, but that's a story for later...